.github/workflows/python_package.yml

```diff
@@ -0,0 +1,89 @@
+name: Python-package
```

Comment:
Please add a badge to README.

Reply:
done in https://github.com/microsoft/LightGBM/pull/3745/commits/1b6f357b33a6df1e0639c66f1952ea17e3350b01